### DISCUSSION TITLE
Write unit tests for devices

### DISCUSSION THREAD

**Discussion by eerison** [Koenkk/zigbee2mqtt#23395] [Devices]
Hello guys

in case you write unit tests for devices definitions, could you post here some examples please, I didn't find any :/

**Comment by Koenkk** [comment]
Definitions itself do not contain logic, but you can write tests for the converters, example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/test/fromZigbee.test.js

**Reply by eerison** [reply]
Thank you <3
